cmd/skywire-cli/commands/hv/hvskychat/root.go

```diff
@@ -0,0 +1,21 @@
+package hvskychat
```

Comment:
I dont think this will ever be used to be honest. Skychat is a test application that we dont ship with the packages.

Reply:
this was just my attempt to evenly implement this functionality for all the UIs.

we have a community member who has a massive rework of skychat in progress.

Otherwise, yes, I think the usage would be slim.